content/projects/linux/_index.md

```diff
@@ -1,6 +1,9 @@
 ---
 title: Linux challenges
+
+ready: true
 ---
+# Beginner
```

Comment:
See what tis looks like when you use `hugo serve`. It's a bit messy. 

Rather `## Beginner` Then `### Task 1 : Basic Task`